lisa/tools/lsmod.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any
+
+from lisa.executable import Tool
+
+
+class Lsmod(Tool):
+    @property
+    def command(self) -> str:
+        return self._command
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self._command = "lsmod"
+
+    def check_loaded_module(
+        self,
+        mod_name: str = "",
+        force_run: bool = False,
+        no_info_log: bool = True,
+        no_error_log: bool = True,
+    ) -> str:
+        if mod_name:
+            self._command = f"lsmod | grep {mod_name}"
```

Comment:
Run the command and get output back to check it. Avoid to use grep. So we can use regexp in the Python code.